panther_analysis_tool/log_schemas/user_defined.py

```diff
@@ -203,6 +203,9 @@ def _extract_schema_name(
                 f" with the prefix '{self._SCHEMA_NAME_PREFIX}'",
             )
 
+        if len(name) > 255:
+            return ("", f"'schema' field: value should not exceed 255 characters")
```

Comment:
linter doesn't like that you're using an f string without variable here

```suggestion
            return ("", "'schema' field: value should not exceed 255 characters")
```